website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh

```diff
@@ -0,0 +1,20 @@
+eksctl create cluster -f - << EOF
```

Comment:
mu concern with this, is that this is now handled by cloudformation and nobody is going to destroy this, leaving garbage in the user account